js/extractRecipeController.js

```diff
@@ -48,15 +49,15 @@ app.controller('extractRecipeController', function($scope) {
         });
     };
     
-    var handleGPU = function(data) {
+    let handleGPU = function(data) {
         $scope.gpuList = data["gpu_list"];
         $scope.canUseGPU = data["can_use_gpu"];
         $scope.gpuUsage = data["gpu_usage_choices"];
-        $scope.gpuMemory = data["gpu_memory_choices"];
+        $scope.gpuMemoryAllocationMode = data["gpu_memory_allocation_mode"];
```

Comment:
Are you sure that this will not break existing recipes? Since there are no migration steps, changing a parameter is a risky operation.